notes/points-markets-settlement.md

```diff
@@ -98,6 +98,18 @@ In the event that the data required by the ***Valuation Algorithm*** are availab
 Note that due to the specifics of oracle protocols, etc. there may be a delay between the conditions for triggering an ***Imminent Airdrop Event*** and the data reaching a market on-chain and the cessation of trading.
 
 
+### Interim Airdrop Event
+
+An ***Interim Airdrop Event*** is deemed to have occurred when some asset with tradable value is issued to at least `Interim Inclusivity Threshold` (%) of all issued *points* (including points issued after the ***Interim Airdrop Event*** snapshot) without *conversion* of the points, i.e. accounts are still deemed to have a certain balance of points assigned to them which could receive further rewards in future.
+
+Once an ***Interim Airdrop Event*** has occurred and all input to the ***Valuation Algorithm*** is available, the *total issued market value* can be calculated. This should be calculated identically to the VWAP in the ***Valuation Algorithm*** for any final token issuance, but in a window starting from the issuance of the ***Interim Airdrop***. Once this price is determined, it can be recorded in a number of ways for future use when a final settlement event occurs:
+
+ - The settlement specification document defined for settlement through UMA could be updated to include a hardcoded value for this VWAP outcome. This would go through Vega Protocol governance to update the market for settlement, but would not go through UMA. The final settlement calculation presented to UMA verifiers would then consist of adding this specified number to the outcome of the final valuation algorithm for the main token.
```

Comment:
This needs updating to be more generic, and to assume all settlement data goes through the oracle. UMA specific options, including what we discussed with @emilbayes and the tactical governance option should move to Appendix I.